include/sch/S_Object/S_Object.h

```diff
@@ -229,7 +229,8 @@ namespace sch
       TSuperellipsoid,
       TSTP_BV_WithPolyhedron,
       T_Point,
-      T_Capsule
+      T_Capsule,
+      TCone
```

Comment:
Why not `T_Cone` for consistency?

Reply:
If you see it carefully it is actually  T_Point,  T_Capsule which are not consistent with the previous return values. In my master branch I fix that, and it was the topic of my next PR.